src/translations/en.json

```diff
@@ -169,6 +169,11 @@
     "cannotPasteTextAsNominalVoltage": "The clipboard text cannot be set as a positive number: ",
     "cannotRetrieveContingencyList": "Could not retrieve contingency list: ",
     "cannotRetrievefilter": "Could not retrieve filter: ",
+    "AddFreePropCrit": "Add a filter by property",
```

Comment:
i think "Ajouter un filtrage par propriété" in EN is "Add filtering by property"

Reply:
Changed